### PR TITLE
#6019 handle list of strs from python

### PR DESCRIPTION
**Pull request by jdoranster**

### CODE REVIEW

**Comment by npilon** [comment]
Can we possibly add some tests for this?

**Comment by jdoranster** [comment]
I think this is ready.

File: pyramid_mustache/tests/test_mustache.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+         {{#title}}{{.}}{{/title}}
+       </h4>
+ {{/Patent}}"""
+
+
+class TestMustache(TestCase):
+
+    def test_mustache_default(self):
+        renderer = Renderer()
+        output = renderer.render(our_template, {'Patent':{'title':'foo-bar','link_number':[u'1234']}})
```
Comment (npilon) [inline pyramid_mustache/tests/test_mustache.py:34]: Do we have test cases for the "just a string" version too?

**Comment by jdoranster** [comment]
Is this sufficient?

**Comment by jdoranster** [comment]
Anything else for this potato?

**Comment by npilon** [comment]
Yeah, update the version of the subpackage in `deus_lex`

**Comment by jdoranster** [comment]
Is this just updating deus_lex/vendor/pyramid-mustache/setup.py?

**Pull request merged**